controllers/eventing/reconciler.go

```diff
@@ -79,8 +79,8 @@ func Reconcile(ctx context.Context, reqLogger logr.Logger, r cloudEventSourceRec
 	}
 
 	if eventSourceChanged {
-		if RequestEventLoop(ctx, reqLogger, r, cloudEventSource) != nil {
-			return ctrl.Result{}, err
+		if errR := RequestEventLoop(ctx, reqLogger, r, cloudEventSource) != nil {
+			return ctrl.Result{}, errR
```

Comment:
Let's reuse the previous var and compare them explicitly
```suggestion
		if err = RequestEventLoop(ctx, reqLogger, r, cloudEventSource); err != nil {
			return ctrl.Result{}, err
```

Reply:
Oh, actually I wrote it wrong before...Thanks for your suggestion, I have corrected it.